src/main/java/hudson/util/VersionNumber.java

```diff
@@ -524,6 +524,14 @@ public boolean isNewerThan(VersionNumber rhs) {
         return compareTo(rhs) > 0;
     }
 
+    public boolean isOlderOrEqualTo(VersionNumber rhs) {
+        return compareTo(rhs) <= 0;
+    }
+
+    public boolean isNewerOrEqualTo(VersionNumber rhs) {
```

Comment:
API methods ought to have included the word `Than`.

Reply:
@jglick yeah, I was a bit uncomfortable with this, and ended with `To` to go with Equal.

Care to confirm so that I avoid another cycle: you meaning `isNewerThanOrEqualTo`?

Reply:
> you meaning `isNewerThanOrEqualTo`?

Yes, that would be better English.

Reply:
Done as https://github.com/jenkinsci/lib-version-number/pull/10